src/map/script.cpp

```diff
@@ -22207,7 +22207,7 @@ static int buildin_mobuseskill_sub(struct block_list *bl,va_list ap)
 	else
 		unit_skilluse_id2(&md->bl, tbl->id, skill_id, skill_lv, casttime, cancel);
 
-	clif_emotion(&md->bl, emotion);
+	clif_emotion(md->bl, static_cast<uint8>(emotion));
 
 	return 1;
 }
```

Comment:
Please copy the validation for the emotion value from BUILDIN_FUNC(emotion) into BUILDIN_FUNC(areamobuseskill)

Reply:
done